pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/AbstractApexNode.java

```diff
@@ -144,4 +146,17 @@ public String getNamespace() {
         // TODO(b/239648780)
         return null;
     }
+
+    /** Returns the string value of the {@link Expression expr}. */
```

Comment:
Without looking at the implementation, I would have assumed that this generates a meaningful String for any sort of Expression. But does it exist to handle a specific case?

I would suggest a clearer name (valueIfStringOrNullLiteral?) and/or more explanation in the javadoc.

Reply:
I originally wanted this function to take all `Expression`s as an input but only produce non-`null` values for `LiteralExpression`s, but I decided to only handle `LiteralExpression`s: [`84342a9`](https://github.com/eklimo/pmd/pull/11/commits/84342a9fc9e51b9ee68d09802da50e477305f815). Please let me know if this is better.

Reply:
Looks good, thanks for the update.